src/express-middleware.js

```diff
@@ -73,13 +73,6 @@ class ExpressMiddleware {
             }
         }
 
-        // nest.js - build request url pattern if exists
-        if (typeof req.params === 'object') {
```

Comment:
i'm 100% with not mutating this object, but i guess we rely on it and it should be handled in the relevant place. i guess when we publish metrics